pytensor/tensor/math.py

```diff
@@ -722,7 +715,25 @@ def isnan(a):
 
 
 def isnan(a):
-    """isnan(a)"""
+    """
+    isnan(a)
+
+    Elementwise test for NaN (not a number).
+
+    This function is a copy of the `isnan` function defined in the
```

Comment:
I don't think we need this detail?
```suggestion
    This function is a copy of the `isnan` function defined in the
```